### ISSUE TITLE
Only sends notifications to expoPushToken generated in Expo Go but fails in standalone app built with EAS build

### ISSUE THREAD

**Issue by jesuscovam** [expo/eas-cli#1249]
### Summary

I'm having issues setting up notifications and at the moment I'm not sure If I setup poorly the configuration for eas build or submit or I'm using the wrong experienceId

I'm building the experience ID this way:
@myusername/myAppSlugInAppJSON

### Managed or bare?

Managed

### Environment

 expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 12.5
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 14.17.1 - ~/.nvm/versions/node/v14.17.1/bin/node
      Yarn: 1.22.17 - ~/.nvm/versions/node/v14.17.1/bin/yarn
      npm: 7.22.0 - ~/.nvm/versions/node/v14.17.1/bin/npm
    Managers:
      CocoaPods: 1.11.3 - /opt/homebrew/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
      Android SDK:
        API Levels: 30, 31
        Build Tools: 30.0.3
        System Images: android-30 | Google APIs Intel x86 Atom, android-31 | ARM 64 v8a
    IDEs:
      Android Studio: 2021.1 AI-211.7628.21.2111.8193401
      Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
    npmGlobalPackages:
      eas-cli: 0.57.0
      expo-cli: 6.0.1
    Expo Workflow: managed

expo doctor:
🎉 Didn't find any issues with the project!

### Error output

<img width="1210" alt="Screen Shot 2022-08-05 at 10 34 23" src="https://user-images.githubusercontent.com/46403187/183111882-53c52782-af1b-4b08-abf8-6c85f9975127.png">

### Reproducible demo or steps to reproduce from a blank project

""

**Comment by dsokal** [comment]
Hi!

It seems it's not an issue with EAS. Please see the docs https://docs.expo.dev/push-notifications/push-notifications-setup/

TLDR:
- `eas credentials`
- Select`Push Notifications: Manage your Apple Push Notifications Key`
- Select `Set up your project to use Push Notifications`
- Follow the instructions display on the screen.

**Comment by jesuscovam** [comment]
Hi, was credentials says the project is already configured, is there something else I could do? @dsokal 
<img width="588" alt="Screen Shot 2022-08-09 at 9 24 51" src="https://user-images.githubusercontent.com/46403187/183674541-d204d814-885d-4a2e-b6d0-538bef1d5b98.png">
